ui/tailwind.config.ts

```diff
@@ -21,14 +21,24 @@ const config = {
             }
         },
         extend: {
+            fontSize: {
+                '2.5': '2.5rem',
+                '1.25': '1.25rem',
+                '1.2': '1.2rem',
+                '1.125': '1.125rem',
+                '1.75':'1.75rem'
+
+            },
```

Comment:
It would be better to use Tailwind's [] feature and use `text-[2.5rem]` instead of extending fontSize. Tailwind has their own scale already that does not follow a number system but rather sm, md, xl, xs, 2xl.